core/streamer.py

```
@@ -295,27 +322,47 @@ def _check():
+                    json.dumps(
+                        {
+                            "type": "headset_status",
+                            "status": status,
+                            "subjectIndex": self.subject_index,
+                            "groupId": self.group_id,
+                            "silentSeconds": round(elapsed),
+                        }
+                    ),
+                )
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)